### PR TITLE
[OKD] release-controller: Add 4-scos-next channel

### PR DESCRIPTION
**Pull request by LorbusChris**
/cc @openshift/okd-maintainers @lmzuccarelli

### CODE REVIEW

**Review by vrutkovs — APPROVED** [review]

**Comment by stbenjam** [comment]
/unassign

Could you get someone from CRT to look? Perhaps @bradmwilliams or someone in #forum-crt.  I have rights but not really the context to what it takes to add a new stream to the release controller.

File: core-services/release-controller/_releases/release-okd-4.y-stable.json [hunk 1]
```diff
@@ -2,7 +2,6 @@
   "name":"4-stable",
   "as": "Stable",
   "message": "All amd64 OKD versions are tracked here",
-  "overrideCLIImage": "image-registry.openshift-image-registry.svc:5000/ocp/4.6:cli",
```
Comment (bradmwilliams) [inline core-services/release-controller/_releases/release-okd-4.y-stable.json:5]: Is there any specific reason that you removed this?
This is a release-controller specific setting that should remain in place.

File: core-services/release-controller/_releases/release-okd-scos-4.y-stable.json [hunk 2]
```diff
@@ -2,7 +2,6 @@
   "name":"4-scos-stable",
   "as": "Stable",
   "message": "All amd64 OKD/SCOS versions are tracked here",
-  "overrideCLIImage": "image-registry.openshift-image-registry.svc:5000/ocp/4.6:cli",
```
Comment (bradmwilliams) [inline core-services/release-controller/_releases/release-okd-scos-4.y-stable.json:5]: Is there any specific reason that you removed this?
This is a release-controller specific setting that should remain in place.

**Review by bradmwilliams — CHANGES_REQUESTED** [review]

File: core-services/release-controller/_releases/release-okd-scos-4.y-stable.json (hunk 2 above)
Comment (LorbusChris) [reply core-services/release-controller/_releases/release-okd-scos-4.y-stable.json:5]: This aligns this config with all the other OKD releases configs - only this one currently has the override.
Are we sure this is still required? 4.6 is EOL

Comment (bradmwilliams) [reply core-services/release-controller/_releases/release-okd-scos-4.y-stable.json:5]: The setting has no real bearing on anything but how the release-controller creates it's releases.  Ultimately, I think it's safe to remove, but I wanted to make sure there wasn't a specific reason that we needed to be aware of.

**Pull request merged**